### DISCUSSION TITLE
Filter text from picture caption

### DISCUSSION THREAD

**Discussion by BenFle11** [aahnik/tgcf#210] [Q&A]
I need help, the bot works fine with forwarding messages, but how can i forward only text messages, i only want to forward text (from a picture with text) to a channel.
Please can anyone help me, i am new with python.

**Comment by tissole** [comment]
I don't think that's possible right now. I tried to whitelist text messages (nofile) and blacklist photos, but it's not working. And probable because text from a photo is not treated as a text message but as a caption. 

You need a new filter: "caption", and an even more specific "caption from a photo" that currently does not exist. There are many requests regarding captions, check #177, maybe this feature will be added too.
